### PR TITLE
Trainee can see classmate

### PR DESCRIPTION
**Pull request by Tranchinhthuc**
Trainee can see members of course

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CoursesController < ApplicationController
+  load_and_authorize_resource
+  before_action :load_course
```
Comment (ThuBM) [inline app/controllers/courses_controller.rb]: only

File: config/routes.rb [hunk 2]
```diff
@@ -32,5 +32,5 @@
   end
 
   resources :users
-
+  resources :courses
```
Comment (ThuBM) [inline config/routes.rb]: only

**Comment by ThuBM** [comment]
ok

File: app/controllers/courses_controller.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CoursesController < ApplicationController
+  load_and_authorize_resource
+  before_action :load_course, only: [:show]
```
Comment (dieunb) [inline app/controllers/courses_controller.rb]: remove, no need

File: app/controllers/courses_controller.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+  def show
+  end
+
+  private
+  def load_course
```
Comment (dieunb) [inline app/controllers/courses_controller.rb]: remove, no need

File: app/controllers/courses_controller.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+class CoursesController < ApplicationController
+  load_and_authorize_resource
+
+  def show
+    @course = Course.find params[:id]
```
Comment (dieunb) [inline app/controllers/courses_controller.rb]: no need

**Pull request merged**